brc.md

```diff
@@ -0,0 +1,92 @@
+# Instructions to Set Up BRC Savio Cluster
+
+## Getting Started with Savio
+
+1. **Create a Savio Account and Request Access**
+   - Visit [https://mybrc.brc.berkeley.edu](https://mybrc.brc.berkeley.edu) to make an account.
+   - Sign the cluster user access agreement.
+   - Request access to `fc_ocow` (Professor Sahai’s project group).
+
+2. **Setup Google Authenticator**
+   - To create one-time passwords for logging in to BRC, follow the instructions [here](https://docs-research-it.berkeley.edu/services/high-performance-computing/user-guide/setting-otp/).
+
+## Accessing BRC via SSH
+
+To access BRC, use the following SSH command:
+```
+ssh yourusername@hpc.brc.berkeley.edu
+```
+
+- At the `Password:` prompt, enter your token PIN followed immediately (without spaces) by the 6-digit one-time password from the Google Authenticator app.
+  - **Example**: If your password is `ilovebrc` and the one-time code is `123456`, you would type `ilovebrc123456`.
```

Comment:
No needed change:
Is "token PIN" (in above bullet) meant to be identical to "password" (in below bullet)?

Reply:
yep, might change that wording a bit